openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -68,14 +68,14 @@ def get_course_card_data(course, org_prefetched=False):
 
     if not org_prefetched:
         course_org = get_organization_by_short_name(course.org)
-        org_description = course_org.get('description')
+        org_description = course_custom_setting.publisher_description or course_org.get('description')
```

Comment:
Add check for blank description values for `course_custom_setting` model.

Reply:
does the blank description include empty spaces or just no value at all?